bin/mv.js

```diff
@@ -127,4 +136,33 @@ function loadProfile(name) {
     var config = new ConfigFile();
 
     return config.getProfile(name);
-}
\ No newline at end of file
+}
+
+function moveCronJob(profile, name, target) {
+    return coroutine(function*() {
+        var cronJobs = yield profile.listCronJobs();
+
+        for (let job of cronJobs) {
+            if (job.name === name) {
+                let token = yield profile.getWebtask({
+                    name: target.name
+                });
+                yield target.profile.createCronJob({
```

Comment:
We will lose all of the cron job's history, but this is unavoidable without working directly with the underlying data stores.